packages/input-amount/src/LionInputAmount.js

```diff
@@ -46,10 +46,28 @@ export class LionInputAmount extends FieldCustomMixin(LocalizeMixin(LionInput))
     super();
     this.parser = parseAmount;
     this.formatter = formatAmount;
+    this.__isPasting = false;
+
+    this.addEventListener('paste', () => {
+      this.__isPasting = true;
+      this.__parserCallcountSincePaste = 0;
+    });
 
     this.defaultValidators.push(new IsNumber());
   }
 
+  __callParser(value = this.formattedValue) {
+    // TODO: input and change events both trigger parsing therefore we need to handle the second parse
```

Comment:
Not sure if this explanation would be clear for everyone. You might consider the elaborate version here, but I'm fine with both.
Wrt the 'TODO:' it doesn't really explain what still needs to be done... :)